### ISSUE TITLE
validate loss cannot update

### ISSUE THREAD

**Issue by Arui1** [khanrc/pt.darts#24]
![image](https://user-images.githubusercontent.com/50826280/65824753-6776e280-e2a0-11e9-911e-52cb78c82e91.png)
loss and prec1 during training is a normal drop, but during  validate ,loss is always some value, prec1 also keep zero. do you met this problem？

**Comment by JiyueWang** [comment]
I have no problem on the cifar10 dataset, but I got the same problem as you when transfer to my own local data. Have you solved the problem?

**Comment by JiyueWang** [comment]
I have solved my problem. The reason is the trainset is not shuffled before split it. Shuffle the indices before split
indices = list(range(n_train))
shuffle = 1
if shuffle:
       np.random.seed(42)
       np.random.shuffle(indices)
train_sampler = torch.utils.data.sampler.SubsetRandomSampler(indices[:split])
